capdrupal.gemspec

```diff
@@ -0,0 +1,33 @@
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+Gem::Specification.new do |spec|
+  spec.name          = 'capdrupal'
+  spec.version       = '8.0.0'
+  spec.authors       = ['Kevin Wenger', 'Yann Lugrin', 'Gilles Doges', 'Toni Fisler', 'Simon Perdrisat', 'Robert Wohleb', 'Kim Pepper']
+  spec.email         = ['dev@antistatique.net']
+
+  spec.description   = <<-DESC
+    A set of tasks for deploying Drupal projects with Capistrano and the help of Drush.
+  DESC
+  spec.summary       = 'A set of tasks for deploying and managing Drupal projects with Capistrano'
+  spec.homepage      = %q{http://github.com/antistatique/capdrupal/}
+  spec.license       = 'MIT'
+
+  # Prevent pushing this gem to RubyGems.org. To allow pushes either set the 'allowed_push_host'
+  # to allow pushing to a single host or delete this section to allow pushing to any host.
+  spec.metadata['allowed_push_host'] = 'NOT ALLOWED'
```

Comment:
We want this gem on RubyGems, so we can remove this?

Reply:
@yannlugrin

Reply:
yes you can remove that